images/installer/Dockerfile.upi.ci

```diff
@@ -78,6 +79,9 @@ RUN pip-3 install cryptography pyOpenSSL==23.2.0
 ENV CLOUDSDK_PYTHON=/usr/bin/python
 ENV CLOUDSDK_PYTHON_SITEPACKAGES=1
 
+# aicli
+RUN pip-3 install -U aicli assisted-service-client
```

Comment:
The problem with this is that the steps might fail when new versions introduce changes. It would be safer to install a specific version (unless you really need always the latest version, of course).